contact.html

```diff
@@ -139,56 +141,53 @@
         <div class="row">
             <div class="col-sm-12 m-auto">
                 <!-- Page Title -->
-                <h1>Contact Us</h1>
+                <h1>Get Touch With Us</h1>
+                <div  style="background-color: red; height: 4px; width: 55px; margin: auto;" class="line">
+
+                </div>
                 <!-- Page Description -->
-                <p> Please contact us with any questions regarding our irrigation system. We're here to help via email, chat, or text.</p>
+                <p style="margin-top: 25px;"> Need to speak to a leading bespoke software development company? We will respond to all your enquires sent to the email address below and via our contact us form within 1 working day </p>
             </div>
         </div>
     </div>
 </section>
 
-<section class="address">
-    <div class="container">
-        <div class="row">
-            <div class="col-lg-5 align-self-center text-center mc-auto">
-                <div class="block">
-                    <div class="address-block text-center mb-5">
-                        <div class="icon">
-                            <i class="fa fa-phone"></i>
-                        </div>
-                        <div class="details">
-                            <p>(+94) 76 893 7675 (Lilan)</p>
-                            <p>Lilan@elzian.com</p>
-                            <p>(+94) 77 295 9306 (Pankajan)</p>
-                            <p>Pankajan@elzian.com</p>
-                        </div>
-                    </div>
-                    <div class="address-block text-center">
-                        <div class="icon">
-                            <i class="fa fa-map-marker"></i>
-                        </div>
-                        <div class="details">
-                            <p>ELZIAN (PVT) LTD,<br>
-                                124/1 ,
-                                Mahawaththa Rd,<br>
-                                Kelaniya, Sri Lanka. </p>
-                        </div>
-                    </div>
-                </div>
+<section class="addres">
+    <div class="containe">
+        <div style="display: flex;" class="">
+            <div style="flex: 0.3; border: 2px solid rgb(238, 237, 237); box-shadow: 4px 4px 2px rgb(238, 237, 237); margin-left: 30px; display: flex; flex-direction: column; align-items: center; justify-content: center;padding: 30px 10px;" >
+
+                <i style="font-size: 48px; color: rgb(238, 57, 57);" class="fas fa-map-marked-alt"></i>
```

Comment:
inline styles needed to remove and added to the style.css

Reply:
@pankajan05 will resolve it asap